src/ovp_pipeline/live_concept_agent.py

```
@@ -0,0 +1,459 @@
+            if pipeline_logger is not None:
+                try:
+                    pipeline_logger.log(
+                        "live_concept_section_write_error",
+                        {
+                            "slug": handle.slug,
+                            "section": section_title,
+                            "error": str(exc),
+                        },
+                    )
+                except Exception:  # noqa: BLE001 — audit best-effort
+                    pass
```

Ruff (0.15.12)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)